lib/bolt/plugin/module.rb

```diff
@@ -267,6 +273,14 @@ def puppet_library(opts, target, apply_prep)
         options = {}
         options[:run_as] = meta_params['_run_as'] if meta_params['_run_as']
 
+        if meta_params['_noop']
```

Comment:
This won't validate parameters for folks using the `task` plugin, this check needs to be there too.